packages/row/src/react/__stories__/index.story.js

```diff
@@ -1,69 +1,66 @@
 import { action } from '@storybook/addon-actions'
-import addons from '@storybook/addons'
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+
```

Comment:
What is the import convention that you're showing here?  Thusfar, my import convention in this project is generally:

named imports, alpha order
blank line
local imports, alpha order
blank line
code...

Reply:
so i guess i've been organizing DS package separately because they kind of feel like a type of local imports...not sure when i started doing that :/

Reply:
I'm open to that.  I guess we need to decide what to do here.